src/other/Paginator.js

```diff
@@ -23,9 +23,38 @@
     Paginator.prototype.publish("numItems", 10, "number", "Pagination total number of items",null,{tags:["Private"]});
     Paginator.prototype.publish("pageNumber", 1, "number", "Pagination set or get the page number",null,{tags:["Private"]});
 
+
+    Paginator.prototype.testData = function() {
+        return this;
+    };
+
     Paginator.prototype.enter = function (domNode, element) {
         HTMLWidget.prototype.enter.apply(this, arguments);
-        this.paginator = element.append("ul").attr("class","paginator pagination pagination-sm");
+        var context = this;
+
+        this.container = element.append("div");
+        this.paginator = this.container.append("ul").attr("class","paginator pagination pagination-sm");
+        this.side = this.container.append("div").attr("class","paginator pagination side");
```

Comment:
Dan's fixedHeader and fixedColumn use **cols-wrapper** and **rows-wrapper**. In your case, **pagination-wrapper** seems appropriate.